src/main/static/stylesheets/variables.scss

```diff
@@ -55,7 +59,7 @@ $width-mobile: 600px;
 
   --background-gradient-footer-light: linear-gradient(180deg, rgba(155, 81, 224, 0) 0%, rgba(155, 81, 224, 0.1) 100%);
   --background-gradient-footer-dark: linear-gradient(180deg, rgba(41, 17, 64, 0) 0%, rgba(41, 17, 64, 0.5) 100%);
-
+  --background-gradient-banner: linear-gradient(180deg, #471671 0%, #371870 100%);
```

Comment:
name this --background-gradient-features-banner, since it's specific to that banner